relayer/chains/cosmos/account.go

```diff
@@ -25,7 +25,7 @@ func (cc *CosmosProvider) GetAccount(clientCtx client.Context, addr sdk.AccAddre
 // GetAccountWithHeight queries for an account given an address. Returns the
 // height of the query with the account. An error is returned if the query
 // or decoding fails.
-func (cc *CosmosProvider) GetAccountWithHeight(clientCtx client.Context, addr sdk.AccAddress) (client.Account, int64, error) {
+func (cc *CosmosProvider) GetAccountWithHeight(_ client.Context, addr sdk.AccAddress) (client.Account, int64, error) {
```

Comment:
It seems like we should use the context here in `queryClient.Account` on line 36 instead of creating anew background context?

Reply:
i actually thought that too but realized the `Account` method expects a normal context whereas the context being passed into `GetAccountWithHeight` is a type from the cosmos sdk.